src/config/dynamic-theme-fixes.config

```diff
@@ -9342,6 +9342,27 @@ CSS
 
 ================================
 
+guildwars2.com
+
+CSS
+body {
+    background-image: none !important;
+}
+body:before {
+    width: 100% !important;
+}
```

Comment:
I'm not sure about this particular change, seems like this isn't necessary. It doesn't obstruct any text.

Reply:
Yes, it's not necessary but that background image on <body> is very bright for dark mode

Reply:
Then we should remove the background image.

Reply:
Do I need to make any changes to this pr?